app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def create
+    post_id = params[:post_id]
+    @comment = Comment.new(user_id: current_user.id, post_id:, text: comment_params[:text])
+
+    if @comment.save
+      puts 'Comment saved successfully!'
+      redirect_to user_post_path(current_user.id, post_id), notice: 'Comment created successfully.'
+    else
+      render :new
+    end
+  end
+
+  def new
+    @post = Comment.new
+  end
```

Comment:
- [ ] Please kindly take note that you are to implement a new comment with `@comment` instance variable and not `@post`. This might generate issues with your project in subsequent milestones, I tried to reach out on slack but did not get a response in time and that's why I have to request for a mandatory change. **The mistake was from the previous reviewer and he had already corrected it. Thank you, sorry for any inconvinience** ℹ️